Document/0x06b-Basic-Security-Testing.md

```diff
@@ -278,6 +278,18 @@ The tool has the following global options (list them via the `show options` comm
 - **SAVE_HISTORY**: If set to "true," the command history will persist across sessions.
 - **VERBOSE, DEBUG**: If set to "true," this will enable verbose and debug logging, respectively.
 
+**Trouble shooting**
+
+Please pay attention, in order to use the modules in Needle. You might have to install its dependencies by using this command in Neelde.
+
+```
+use device/dependency_installer
+
+run
+```
+Other module may promt you the ```apt-get``` command has not been installed. To get ```apt-get```, go to your Cydia and look for  ```CyDelete``` and install it. 
```

Comment:
`promt` should be `prompt`. 

> look for  ```CyDelete``` and install it

Is this correct? CyDelete looks to me like something that deletes things, not something that installs things.

Reply:
You'r correct. But in order to install ``apt-get``` command, this package ```CyDelete``` will install it for us. Well, it could be other ways to get the ```apt-get``` installed though. For me, installing ```CyDelete``` is the fastest way.